diff -r a42732dfd9a0 qtgui/main.cpp
--- a/qtgui/main.cpp	Thu Sep 22 14:01:55 2016 +0200
+++ b/qtgui/main.cpp	Sat Sep 24 16:46:54 2016 +0200
@@ -34,7 +34,10 @@
 	QFont::insertSubstitution(".Helvetica Neue DeskInterface", "Lucida Grande");
 	QFont::insertSubstitution(".SF NS Text", "Lucida Grande");
 #endif
-	
+#if defined(Q_WS_X11)
+	QApplication::setDesktopSettingsAware(false);
+	QApplication::setStyle("gtk"); // "gtk, plastique, cleanlooks, cde, ..."
+#endif	
 	LuxGuiApp application(argc, argv);
 
 /*	QString locale = QLocale::system().name();
